src/CollectBlock.ts

```diff
@@ -61,7 +60,7 @@ const equipToolOptions = {
 }
 
 async function mineBlock (bot: Bot, block: Block, options: CollectOptionsFull): Promise<void> {
-  if (bot.blockAt(block.position) !== block) {
```

Comment:
Sorry, I broke this in #84 , but I've also fixed it in #88. When merging make sure only one of the changes will stay in the code.